geofileops/geoops.py

```diff
@@ -100,12 +100,8 @@ def dissolve_within_distance(
     nb_steps = 9
 
     # Already check here if it is useful to continue
-    if output_path.exists():
-        if force is False:
-            logger.info(f"Stop, output exists already {output_path}")
-            return
-        else:
-            fileops.remove(output_path)
+    if _io_util.output_exists(path=output_path, force=force):
+        return True
```

Comment:
```suggestion
        return
```
This function doesn't return a value... so (like the original code) it shouldn't return True... There are quite some cases like this further on as well...